curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f1.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-41
 
 # --description--
 
-You want to add the same gradient to the next two sections. Instead of doing that, create a new class called `bb1-window`, and move the `height` and `background` properties and values from `.bb1a` to the new class.
+You want to add the same gradient to the next two sections. Instead of doing that, create a new class selector called `bb1-window`, and move the `height` and `background` properties and values from `.bb1a` to the new class selector.
 
 # --hints--
 
```

Comment:
I can't quite reach it, but the hint text right below this line says "class declaration" - we should change that to "class selector" too, I think.

Reply:
I think, you are right. So here is a commit for it.